config.json

```diff
@@ -1,10 +1,176 @@
 {
-  "track_id": "clojurescript",
-  "language": "ClojureScript",
-  "active": false,
-  "blurb": "",
-  "ignore_pattern": "[Ee]xample",
-  "solution_pattern": "[Ee]xample",
-  "test_pattern": "[Tt]est",
-  "exercises": null
+    "language": "ClojureScript",
+    "track_id": "clojurescript",
+    "active": true,
+    "blurb": "",
+    "test_pattern": "[Tt]est",
+    "solution_pattern": "[Ee]xample",
+    "ignore_pattern": "[Ee]xample",
+    "exercises": [
+        {
+            "topics": null,
+            "difficulty": 1,
+            "unlocked_by": null,
+            "auto_approve": true,
+            "core": false,
```

Comment:
@iHiD I don't know exactly what happens if all the exercises have `core` set to `false`. Do you? We probably don't want to make too much of an effort to order the exercise with unlocks and such, right? Or maybe a small pass.